.github/workflows/build.yml

```diff
@@ -0,0 +1,53 @@
+name: Build
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        target: [
+            aarch64-unknown-linux-gnu,
+            x86_64-linux-android,
```

Comment:
Might want to comment this one out until https://github.com/xmtp/libxmtp/issues/134 is addressed, if we want to merge with 🟢 CI anyway